src/app/components/BibPage/BibPage.jsx

```diff
@@ -80,8 +77,7 @@ const BibPage = (props) => {
                   shortenItems={shortenItems}
                   items={items}
                   bibId={bibId}
-                  title={`${bib.numItems} item${bib.numItems === 1 ? '' : 's'}
-                    associated with this record:`}
+                  title="AVAILABILITY"
```

Comment:
Yeah we do but it's assuming the entire page is laid out in a data table, which we haven't done yet, so it is a little funny right now.